pages/about.tsx

```diff
@@ -64,7 +64,7 @@ const AboutPage = () => {
           </blockquote>
         </div>
       </div>
-      <div className="w-2/3 relative flex items-center justify-center">
+      <div className="w-full sm:w-2/3 relative sm:flex sm:items-center sm:justify-center">
```

Comment:
the diagram isn't actually showing up on mobile yet - it's a tricky scenario where the parent of the Next `Image` needs a position which is then interfering with the flex/flex-col up on L6. 

ideas/edits welcome please!

Reply:
Yeah I also hit this issue, where a Next `Image` is always wrapped in it's own div, so doesn't behave as you'd expect.

We could try to import `next/future/image` which resolves this (and is how images are handled in v13 onwards).

Happy to have a look at this and try to see what I can come up with as is though!

Reply:
thanks for this tip - swapped our import for `next/future/image` and honestly just got way better console warnings/suggestions :sweat_smile: 

diagram margins aren't quite as clean/impactful on mobile yet but i think it'll do for mvp!
![Screenshot from 2023-01-26 22-02-26](https://user-images.githubusercontent.com/5132349/214950024-183f6447-6605-4c4e-a770-b52b1577bd66.png)